spec/yaml/swiftnav/sbp/system.yaml

```diff
@@ -215,6 +214,59 @@ definitions:
             fill: SubSystemReport
             desc: Reported status of individual subsystems
 
+ - StatusJournalItem:
+    short_desc: Subsystem Status report
+    embedded_type: true
+    desc: >
+      Reports the uptime and the state of a subsystem via generic and specific
+      status codes.  If the generic state is reported as initializing, the
+      specific state should be ignored.
+    fields:
+        - uptime:
+            type: u32
+            desc: Milliseconds since system startup
+        - report:
+            type: SubSystemReport
+
+ - MSG_STATUS_JOURNAL:
+    id: 0xFFFD
+    short_desc: Status report journal
+    desc: >
+        The status journal message contains past status reports (see
+        MSG_STATUS_REPORT) and functions as a error/event storage for telemetry
+        purposes.
+    fields:
+        - reporting_system:
+            type: u16
+            desc: Identity of reporting system
+            fields:
+              - 0-15:
+                 desc: System
+                 values:
+                     - 0: Starling
+                     - 1: Precision GNSS Module (PGM)
+        - sbp_version:
+            type: u16
+            desc: SBP protocol version
+            fields:
+              - 8-15:
+                  desc: SBP major protocol version number
+              - 0-7:
+                  desc: SBP minor protocol version number
+        - total_status_reports:
+            type: u32
+            desc: Total number of status reports sent since system startup
+        - sequence_descriptor:
```

Comment:
The `n_...` convention is overloaded with lots of different semantic meanings (we also generate `n_...` fields programmatically for variable length arrays)... so just picked something new here that has cleaner mapping between it's English meaning (and common usage in APIs) and it's semantics for the message.  Also reduced the index/count to a combo field with 4 bits for the count and 4 bits for the index, which is the same as the `n_obs` field for `MsgObs` -- this will allow us to send 30*15 (450) "journal items" in one sequence, since we're currently only sending one item in the sequence this seemed sufficient to satisfy long term expansion.